mod_auth/forms.py

```diff
@@ -7,13 +7,24 @@
 
 
 def unique_username(form, field):
-    # Check if a user already exists with this name
+    	"""
```

Comment:
Alignment is still not correct... Compare to the line with the comment that's gone now...